content/docs/replication/high-availability/powerstore-metro.md

```diff
@@ -45,7 +45,7 @@ volumeBindingMode: Immediate
 When a Metro `PV` is created, the volumeHandle will have the format `<volumeID/globalID/protocol:remote-volumeID/remote-globalID>`.
 
 ### PowerStore Metro volume expansion
-When a request is made to increase the size of a Metro `PV`, the metro replication session is temporarily paused. The size of the local/preferred volume is then increased. After the size of the local/preferred volume has been updated, the metro session is resumed. It's important to note that the paths for the remote/non-preferred volume will not become active until the metro session is resumed and the remote/non-preferred volume reflects the updated size.
+When a request is made to increase the size of a Metro `PV`, the metro replication session must be temporarily paused prior to the editing of Kubernetes resources. This can be done from the PowerStore Web UI or CLI. The size of the local/preferred volume is then increased. The metro session is then manually resumed. It's important to note that the paths for the remote/non-preferred volume will not become active until the metro session is resumed and the remote/non-preferred volume reflects the updated size.
```

Comment:
Should we also say "metro session must be then manually resumed" to align with the initial pause statement? Just for clarity and consistency.  @sharmilarama  let me know if you agree.

Reply:
I thought I already mentioned it,  "The metro session is then manually resumed."

Reply:
Yes, my suggestion was around wording and giving a clear customer directive.